src/cli/commands/init.js

```diff
@@ -114,14 +121,18 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       def = val;
     }
 
+    if (manifestKey === 'private' && privateFlag) {
+      def = true;
+    }
+
     if (def) {
-      question += ` (${def})`;
+      question += ` (${def.toString()})`;
     }
 
     let answer;
     let validAnswer = false;
 
-    if (yes) {
+    if (yes || (privateFlag && manifestKey === 'private')) {
```

Comment:
I think this additional `||` would also be eliminated if the `default` was set based ont he flag in the array of questions.